apps/openmw/mwworld/store.cpp

```diff
@@ -179,16 +185,19 @@ namespace MWWorld
         return ptr;
     }
     template<typename T>
-    void Store<T>::load(ESM::ESMReader &esm, const std::string &id) 
+    RecordId Store<T>::load(ESM::ESMReader &esm) 
     {
-        std::string idLower = Misc::StringUtils::lowerCase(id);
+        T record;
+        bool isDeleted = false;
 
-        std::pair<typename Static::iterator, bool> inserted = mStatic.insert(std::make_pair(idLower, T()));
+        record.load(esm, isDeleted);
+        Misc::StringUtils::toLower(record.mId);
+
+        std::pair<typename Static::iterator, bool> inserted = mStatic.insert(std::make_pair(record.mId, record));
         if (inserted.second)
             mShared.push_back(&inserted.first->second);
 
-        inserted.first->second.mId = idLower;
-        inserted.first->second.load(esm);
```

Comment:
You mean like only modify a subrecord in an addon and leave the rest of the original record unchanged? Sounds useful, but also very complicated. Definitely not on my list for the near or medium future. OpenMW 2.0-ish? I don't think we need to take that into consideration now.